ocspchecker/ocspchecker.py

```diff
@@ -61,14 +63,10 @@ class OcspResponderError(Exception):
     "140070EF": "Unable to find public key parameters.",
 }
 
+def get_ocsp_status(host: str, port: int = 443, proxy: Union[None, Tuple[str, int]] = None, request_timeout: float = 3.0) -> List[str]:
```

Comment:
I think the only other feedback I have is why did you hard code the port?  I made it configurable should https be listening on a different port.  If left blank, it would default to 443.

Reply:
The only difference is that now you can't call `get_ocsp_status('some.host.com', None)` anymore, but this way 443 is the default port (if not provided) and it's default value is visible from IDE's autocomplete & doc.

A few examples:
```python
get_ocsp_status('some.host.com', None)   # Now broken
get_ocsp_status('some.host.com')         # Same as before, port=443
get_ocsp_status('some.host.com', 80)     # Same as before, port=80
```

I didn't realize that is breaking the first use case, although the second one is semantically less weird and I would have pick that one. I can undo that change if you feel like it

Reply:
I'm going to accept this as is and do more testing / noodling on this.  I can always just change it back.